packages/sdks/typescript/src/tests/common/chat.test.ts

```diff
@@ -0,0 +1,177 @@
+import { LogSources } from '@latitude-data/core/browser'
+import {
+  mock502Response,
+  mockAuthHeader,
+  mockBodyResponse,
+  mockChatMessage,
+} from '$sdk/tests/common/helpers'
+import { setupServer } from 'msw/node'
+import {
+  afterAll,
+  afterEach,
+  beforeAll,
+  describe,
+  expect,
+  it,
+  vi,
+} from 'vitest'
+
+import { Latitude } from '../../index'
+import { parseSSE } from '../../utils/parseSSE'
+
+let FAKE_LATITUDE_SDK_KEY = 'fake-api-key'
+let sdk: Latitude
+
+const server = setupServer()
+
+describe('/chat', () => {
+  beforeAll(() => server.listen())
+  afterEach(() => server.resetHandlers())
+  afterAll(() => server.close())
+
+  describe('v1', () => {
+    beforeAll(() => {
+      sdk = new Latitude(FAKE_LATITUDE_SDK_KEY, {
+        __internal: {
+          retryMs: 10,
```

Comment:
Faster test. We were spending 3 seconds in the retry test. Now I do twice one by api version